test/ModuleLoader.test.cpp

```diff
@@ -43,6 +55,10 @@ TEST(ModuleLoader, InternalModulePassed)
 
 	EXPECT_EQ("IPDropDown", moduleManager.GetModuleName(0));
 
+	// Ensure ModuleManager does not allow multiple modules with same name but different casing
+	EXPECT_NO_THROW(moduleManager.RegisterModule(static_cast<std::unique_ptr<GameModule>>(std::make_unique<DifferentCasedNameModule>())));
```

Comment:
I believe this should be `EXPECT_THROW`, with the exception type listed.

Reply:
I don't think we should create an exception for adding 2 modules with the same name. We would have to catch the exception to keep Outpost 2 from closing. I think notifying the user of the issue and logging is sufficient.

Reply:
Ahh, ok, I understand. I would be tempted to catch the exception and translate it to a log message, but it depends on how deep down the code is. I don't really like having to catch exceptions directly around where they are generated.